### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by kafkas** [latitudegames/GPT-3-Encoder#7]
Can we please get a new release incorporating the recent changes?

**Comment by spencercap** [comment]
+1
the npm distribution isnt current w the repo's project files. 

temp solve: 
```json
"dependencies": {
    "gpt-3-encoder": "latitudegames/GPT-3-Encoder"
    ...
}
```

**Comment by seanulacra** [comment]
Hi all, sorry for the delay - just published 1.1.4 so I'll close this issue now. Thanks for your patience!
